beam/README.md

```diff
@@ -8,12 +8,9 @@ Current interpreter implementation supports the static repl. It compiles the cod
 You have to first build the Beam interpreter by enable the **beam** profile as follows:
 
 ```
-mvn clean package -Pbeam -DskipTests
+mvn clean package -Pbeam -DskipTests -Pscala-2.10
```

Comment:
@XuMingmin Hi Zeppelin can be built with scala-2.11

- https://github.com/apache/zeppelin/blob/master/dev/change_scala_version.sh

You can use it like

```
./dev/change_scala_version.sh 2.11
mvn clean package...
`